src/TraceEvent/TraceLog.cs

```diff
@@ -7125,6 +7131,14 @@ internal TraceModuleFile UniversalMapping(ProcessMappingTraceData data, ProcessM
                 
                 InsertAndSetOverlap(index + 1, loadedModule);
             }
+            else
```

Comment:
This `else` clause is valid if the `ImageBase` of `loadedModule` matches the `StartAddress` of the new mapping.  But, in this case, we still should update `loadTimeQPC` to match the new mapping timestamp.  We also need to determine if `overlaps` needs to be set because we're not re-inserting it.  If we can't do this here, we will need to just create a new `TraceLoadedModule` and insert it so that the underlying code can properly set `overlaps` if necessary.